### PR TITLE
Pk/update starlark lsp readme

### PR DESCRIPTION
**Pull request by Peeeekay**

### CODE REVIEW

File: starlark-lsp/README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # starlark lsp
 
-A language server for starlark based kurtosis packages, this is forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) and
-extended to support kurtosis builtin types and instructions. 
+It is a language server for Starlark with customized Kurtosis specific intellisense. You can learn more about language server here and learn more about Starlark here.
+
+
+## Acknowledgments
+This `starlark-lsp` is intially forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) that contains Apache 2.0 liscense. It has been extended to support kurtosis builtins.
```
Comment (h4ck3rk3y) [inline starlark-lsp/README.md]: s/is/was/g

s/that contains Apache 2.0 liscense/which is licensed as Apache 2.0 liscence/g 

s/kurtosis/Kurtosis

File: starlark-lsp/README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # starlark lsp
 
-A language server for starlark based kurtosis packages, this is forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) and
-extended to support kurtosis builtin types and instructions. 
+It is a language server for Starlark with customized Kurtosis specific intellisense. You can learn more about language server here and learn more about Starlark here.
```
Comment (h4ck3rk3y) [inline starlark-lsp/README.md]: where is "here"?

Comment (h4ck3rk3y) [reply starlark-lsp/README.md]: s/about language/about the language/g

**Review by h4ck3rk3y — APPROVED** [review]

File: starlark-lsp/README.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
 # starlark lsp
```
Comment (h4ck3rk3y) [inline starlark-lsp/README.md]: Starlark

File: starlark-lsp/README.md [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
-A language server for starlark based kurtosis packages, this is forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) and
-extended to support kurtosis builtin types and instructions. 
-
+# Starlark Language Server
 
+It is a language server for Starlark with customized Kurtosis specific intellisense. You can learn more about about the Language Server Protocol(LSP) [here](https://langserver.org/) and learn more about Starlark [here](https://bazel.build/rules/language).
 
 
+## Acknowledgments
+This `starlark-lsp` was intially forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) which is licensed as Apache 2.0 liscence. It has been extended to support Kurtosis builtins.
```
Comment (leeederek) [inline starlark-lsp/README.md]:
```suggestion
This `starlark-lsp` was intially forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) which is licensed as Apache 2.0 license. The `starlark-lsp` has been extended to support Kurtosis built-ins.
```

**Pull request merged**